src/sql/parts/jobManagement/common/agentJobUtilities.ts

```diff
@@ -51,4 +51,41 @@ export class AgentJobUtilities {
 			return date;
 		}
 	}
+
+	public static setRunnable(icon: HTMLElement, index: number) {
+		if (icon.className.includes('non-runnable')) {
+			icon.className = icon.className.slice(0, index);
+		}
+	}
+
+	public static getActionIconClassName(startIcon: HTMLElement, stopIcon: HTMLElement, executionStatus: number) {
+		this.setRunnable(startIcon, 9);
```

Comment:
I think `8` and `9` are the lengths of some portion of a class name?  is there some way to compute these values so we don't get breaks if someone changes class names, etc?  

If that's not what these numbers are for can we add a comment where they came from?